test/extended/cluster/utils.go

```diff
@@ -0,0 +1,347 @@
+package cluster
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+	"unicode"
+
+	"github.com/fatih/structs"
```

Comment:
We don't use submodules - is fatih already included in our vendor tree?  How hard is it to remove this dependency?